tests/testthat/test-5transformCounts.R

```diff
@@ -15,7 +15,7 @@ test_that("relabundance", {
     f <- rev(seq_len(ncol(relabundance(actual))))
     relabundance(actual) <- relabundance(actual)[,f]
     expect_equal(rel_mat,
-                 relabundance(actual)[,f])
+                 relabundance(actual), check.attributes = FALSE)
 })
```

Comment:
We have this function called relabundance... I think it does not add any value; should we remove it?

It just fetches relative abundance table from assay. It does not calculate anything

Exactly same as assay(tse, "relabundance")

Reply:
I agree. I always thought it is a bit confusing.

But it should be deprecated and not removed immediately so that users have time to adjust their workflows.